CHANGES.txt

```diff
@@ -5,6 +5,9 @@ Changes for Crate Data JDBC Client
  Unreleased
  ==========
 
+ - Fix the issue with the crate jdbc driver failing when an other
```

Comment:
fixed an issue wich caused the crate driver to take over jdbc urls from none crate urls